src/components/code.tsx

```
@@ -0,0 +1,71 @@
+          style={{
+            fontSize: 30,
+            color: "#d1d5db",
+            marginBottom: 20,
+            display: "flex",
+          }}
+        >
+          Language: {"Typescript"}
+        </div>
+        <div>
+          <pre>
+            <code dangerouslySetInnerHTML={{ __html: highlightedCode }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)